### PR TITLE
Testnet userid page layout fixes for small screens

### PR DESCRIPTION
**Pull request by manjash**
## Summary
For small screens:
- Font size changed for graffiti and summary numbers
- Fish icon is smaller
- Positioning of the page start is fixed
- Large graffiti strings now break
- Fixed horizontal scroll issue 

https://github.com/iron-fish/website-testnet/issues/135

## Testing Plan
Used mock data in apiClient/client.ts.
Tested on desktop Chrome  browser (responsive regime, several mobile devices) and with Safari on iPhone 13 (landscape and portrait orientations).

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

**Comment by NullSoldier** [comment]
Looks like this fails to build.

**Comment by brekk** [comment]
@manjash Looks like there's an area where you copy-pasted raw SVG (?), you might need to change `class` to `className`. You can also run the linting locally and that should show you the issue as well. `nps lint` / see `package-scripts.js` for more.

**Comment by manjash** [comment]
@brekk Hey, thank you very much for the hints :)

**Comment by manjash** [comment]
@dguenther Hey, is this a problem on my end again?

**Comment by dguenther** [comment]
Nope, I'm just looking through this now. Thanks for asking though!

File: components/OffsetBorder/Box.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export function Box({
         `relative`,
         `mb-2.5`,
         container ? `ml-2.5` : ``,
-        `mr-2.5`,
+        `mr-5`,
```
Comment (dguenther) [inline components/OffsetBorder/Box.tsx:17]: This margin value is specifically chosen to offset the `-right-2.5` values in the enclosed div. We should leave this as it was at `mr-2.5`

**Review by dguenther — CHANGES_REQUESTED** [review]
Left a few change requests -- In general, I'd say this PR should aim for the desktop / 1920x1080 size to look exactly the same as it does now. (aside from copy changes like "All Time Rank" to "Rank"). It'll help if you try to make the PR as small as possible, for example, just adding a breakpoint for smaller header text along with `break-all` will solve most of the small screen issues.

File: pages/users/[id].tsx [hunk 2]
```diff
@@ -182,44 +182,48 @@ export default function User({ loginContext }: Props) {
       <main className="bg-ifpink flex-1 justify-center flex pt-16 pb-32">
         <div style={{ flexBasis: 1138 }}>
           <OffsetBorderContainer>
-            <div className="px-24 pt-16 pb-12">
+            <div className="px-7 pt-7 md:px-24 md:pt-16 pb-12">
```
Comment (manjash) [inline pages/users/[id].tsx:185]: I hope you don't mind that I left 2 different sets of paddings (fixed the pt for desktop though), it's more aligned with the designs (and saves space on mobile)

File: pages/users/[id].tsx [hunk 3]
```diff
@@ -249,7 +253,7 @@ export default function User({ loginContext }: Props) {
                       <tr className="text-xs text-left tracking-widest border-b border-black">
                         <th className="font-normal py-4">ACTIVITY</th>
                         <th className="font-normal">DATE</th>
-                        <th className="font-normal">POINTS</th>
+                        <th className="font-normal pr-2">POINTS</th>
```
Comment (manjash) [inline pages/users/[id].tsx:256]: Beautification mostly, it was almost spaceless on mobile (POINTSDETAILS)

**Comment by dguenther** [comment]
We ended up implementing this with some additional feedback from our designer and deployed it out recently -- Sorry for failing to respond to your comments sooner here. I'm going to close this PR, but you're still welcome to submit it for testnet credit if you haven't already!

**Pull request closed without merging**